library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+    * Abnormal-interictal-rhythmic-activity <nowiki> {suggestedTag=Rhythmic-activity-attribute, suggestedTag=Polymorphic-delta-activity-morphology, suggestedTag=Frontal-intermittent-rhythmic-delta-activity-morphology, suggestedTag=Occipital-intermittent-rhythmic-delta-activity-morphology, suggestedTag=Temporal-intermittent-rhythmic-delta-activity-morphology, suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern, suggestedTag=Finding-incidence}[Activity of frequency lower than alpha, that clearly exceeds the amount considered physiologically normal for the patients age and state of alertness. (Source: Beniczky ea 2013, Appendix S4; Morphologies from Beniczky ea 2017, Table 5; Suggested tags from Beniczky ea 2017, Section 8.)] </nowiki>
+    * Interictal-special-patterns <nowiki> [(Source: Beniczky ea 2017, Table 5.)]</nowiki>
+        ** Interictal-periodic-discharges <nowiki> {suggestedTag=Periodic-discharge-morphology, suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=RPP-time-related-features} [Periodic discharge not further specified (PDs). (Source: Beniczky ea 2017, Table 5.)]</nowiki>
+            *** Generalized-periodic-discharges <nowiki> [GPDs. The term generalized refers to any bilateral, bisynchronous and symmetric pattern, even if it has a restricted field (e.g. bifrontal). (Source: Beniczky ea 2017, Table 5; Hirsch ea 2013.) ]</nowiki>
+            *** Lateralized-periodic-discharges <nowiki> [LPDs. Lateralized includes unilateral and bilateral synchronous but asymmetric; includes focal, regional and hemispheric patterns. (Source: Beniczky ea 2017, Table 5; Hirsch ea 2013.)]</nowiki>
+            *** Bilateral-independent-periodic-discharges <nowiki> [BIPDs. Bilateral Independent refers to the presence of 2 independent (asynchronous) lateralized patterns, one in each hemisphere. (Source: Beniczky ea 2017, Table 5; Hirsch ea 2013.)]</nowiki>
+            *** Multifocal-periodic-discharges <nowiki> [MfPDs. Multifocal refers to the presence of at least three independent lateralized patterns with at least one in each hemisphere. (Source: Beniczky ea 2017, Table 5; Hirsch ea 2013.)]</nowiki>
+        ** Extreme-delta-brush <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern}[(Source: Beniczky ea 2017, Table 5.)]</nowiki>
+
+
+'''Critically-ill-patients-patterns''' <nowiki> [Rhythmic or periodic patterns in critically ill patients (RPPs) are scored according to the 2012 version of the American Clinical Neurophysiology Society Standardized Critical Care EEG Terminology (Source: Hirsch ea 2013; Beniczky ea 2017, Section 9).]</nowiki>
```

Comment:
Should this be Critical-ill-patient-patterns?

Reply:
sure, fixed